src/sentry/migrations/0126_backfill_platformexternalissue_project_id.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-11-08 04:14
+from __future__ import unicode_literals
+
+from django.db import migrations
+import django.db.models.deletion
+import sentry.db.models.fields.foreignkey
+from sentry.models.platformexternalissue import PlatformExternalIssue
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_platformexternalissue_project_id(apps, schema_editor):
+    """
+    Fill the PlatformExternalIssue.project_id from related Group.project_id.
+    """
+    PlatformExternalIssue = apps.get_model("sentry", "PlatformExternalIssue")
+    all_external_issues = PlatformExternalIssue.objects.filter(project_id__isnull=True).select_related("group")
```

Comment:
This is where the issue manifests.... "all" is a misnomer if every group is not guaranteed to exist... so we're only doing a partial backfill where a backfill is possible.